src/Sql/Sql/ManagedDatabase Backup/Cmdlet/RemoveAzureSqlManagedDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,236 @@
+        /// </summary>
+        public SwitchParameter PassThru { get; set; }
+
+        /// <summary>
+        /// Returns true if the model object that was constructed by this cmdlet should be written out
+        /// </summary>
+        protected override bool WriteResult() { return PassThru; }
+
+        /// <summary>
+        /// Get the entities from the service
+        /// </summary>
+        /// <returns>The list of entities</returns>
+        protected override IEnumerable<AzureSqlManagedDatabaseLongTermRetentionBackupModel> GetEntity()
+        {
+            return ModelAdapter.GetManagedDatabaseLongTermRetentionBackups(
+                Location,
+                ManagedInstanceName,
+                DatabaseName,
+                BackupName,
+                ResourceGroupName,
+                null,
+                null);
+        }
+
+        /// <summary>
+        /// No user input to apply to model
+        /// </summary>
+        /// <param name="model">Model retrieved from service</param>
+        /// <returns>The model that was passed in</returns>
+        protected override IEnumerable<AzureSqlManagedDatabaseLongTermRetentionBackupModel> ApplyUserInputToModel(
+            IEnumerable<AzureSqlManagedDatabaseLongTermRetentionBackupModel> model)
+        {
+            return model;
+        }
+
+        /// <summary>
+        /// No changes to persist to server
+        /// </summary>
+        /// <param name="entity">The output of apply user input to model</param>
+        /// <returns>The input entity</returns>
+        protected override IEnumerable<AzureSqlManagedDatabaseLongTermRetentionBackupModel> PersistChanges(
+            IEnumerable<AzureSqlManagedDatabaseLongTermRetentionBackupModel> entity)
+        {
+            ModelAdapter.RemoveManagedDatabaseLongTermRetentionBackup(Location, ManagedInstanceName, DatabaseName, BackupName, ResourceGroupName);
+            return entity;
+        }
+
+        /// <summary>
+        /// Entry point for the cmdlet
+        /// </summary>
+        public override void ExecuteCmdlet()
+        {
+            if (InputObject != null)
+            {
+                Location = InputObject.Location;
+                ManagedInstanceName = InputObject.ServerName;
+                DatabaseName = InputObject.DatabaseName;
+                BackupName = InputObject.BackupName;
+                ResourceGroupName = InputObject.ResourceGroupName;
+            }
+            else if (!string.IsNullOrWhiteSpace(ResourceId))
+            {
+                ParseLongTermRentionBackupResourceId(ResourceId);
+            }
+
+            if (ShouldProcess(this.BackupName))
+            {
+                if (Force.IsPresent || ShouldContinue(
+                    string.Format(CultureInfo.InvariantCulture, Properties.Resources.RemoveAzureSqlDatabaseLongTermRetentionBackupDescription, this.BackupName, this.DatabaseName, this.ManagedInstanceName, this.Location),
```

Comment:
I think you need a new constant (MI version) for RemoveAzureSqlDatabaseLongTermRetentionBackupDescription
RemoveAzureSqlDatabaseLongTermRetentionBackupWarning,

Please check all other places